app/components/ChallengeDetailComponents/ChallengeStartView.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+  Button,
+  StyleSheet,
+  TouchableHighlight,
+  Image
+} from 'react-native';
+
+import { MainMapRoute } from '../../routes/defaultRoutes'
+import run from '../../assets/run.png';
+
+export default class StartChallenge extends Component {
+  constructor(props) {
+    super(props);
+    this.goToMainMapScene = this.goToMainMapScene.bind(this);
+  }
+  goToMainMapScene(challenge) {
+    var selectedLocation = challenge; // TEMP MOCK DATA
```

Comment:
unless there's a reason to save this variable, I would just use `challenge` directly.